build/yamls/elk-flow-collector/deploy.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
Lets keep the script at hack/elk-flow-collector/deploy.sh?

Reply:
In addition, could we change the network visibility documentation where there are steps to deploy elk-flow-collector? Using this script should make the deployment steps easy.

Reply:
Sure, thanks for reminding.